### PR TITLE
Fix ha compute scenario configuration.

### PR DESCRIPTION
**Pull request by gosipyan**

### CODE REVIEW

File: jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml [hunk 1]
```diff
@@ -169,7 +169,7 @@
           export scenario=\"/root/scenario.yml\" ;
           export commands=\"$commands\" "'
 
-
+          sed -i -e "s,##shared_nfs_for_glance##,$shared_storage_ip," scenario.yml
```
Comment (dirkmueller) [inline jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml]: this should be also including a macro for the mount path:

```bash
sed -i -e "s,##shared_nfs_export_for_glance##,/var/$cloud/ha-glance/," scenario.yml
```

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml [hunk 2]
```diff
@@ -1,6 +1,22 @@
 ---
 # 3 - 7 nodes: HA IMPI 1 x 3, KVM x 3
 proposals:
+- barclamp: nfs_client
+  name: glance
+  attributes:
+    exports:
+      glance:
+        nfs_server: ##shared_nfs_for_glance##
+        export: "/var/qa4/ha-glance/"
```
Comment (dirkmueller) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: needs to be

```
##shared_nfs_export_for_glance##
```

File: scripts/scenarios/cloud7/qa/ssl-insecure/qa-scenario-3.yaml [hunk 3]
```diff
@@ -1,6 +1,23 @@
 ---
 # 3 - 7 nodes: HA IMPI 1 x 3, KVM x 3
 proposals:
+- barclamp: nfs_client
+  name: glance
+  attributes:
+    exports:
+      glance:
+        nfs_server: ##shared_nfs_for_glance##
+        export: "/var/qa4/ha-glance/"
```
Comment (dirkmueller) [inline scripts/scenarios/cloud7/qa/ssl-insecure/qa-scenario-3.yaml]: same

File: jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml (hunk 1 above)
Comment (gosipyan) [inline jenkins/ci.suse.de/cloud-mkphyscloud-qa-scenario-3.yaml]: Fixed

File: scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml (hunk 2 above)
Comment (gosipyan) [inline scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-3.yaml]: Fixed

File: scripts/scenarios/cloud7/qa/ssl-insecure/qa-scenario-3.yaml (hunk 3 above)
Comment (gosipyan) [inline scripts/scenarios/cloud7/qa/ssl-insecure/qa-scenario-3.yaml]: Fixed

**Comment by gosipyan** [comment]
@nicolasbock not sure what you mean :(

**Comment by nicolasbock** [comment]
@gosipyan sorry I should have explained this better: The email address in the commit is wrong. You probably haven't set

~~~
git config user.email Gayane.Osipyan@suse.com
~~~

Once you set your email, you can reset it in the commit by running

~~~
git commit --amend --reset-author
~~~

Then Github is able again to correctly associate this commit with you :smile:.

**Comment by gosipyan** [comment]
@nicolasbock Many thanks for noticing that !!! Done

**Comment by nicolasbock** [comment]
@gosipyan If you like you can also add your suse email address to your GitHub profile so that GitHub gives you proper attribution :smile:.

**Pull request merged**